src/sass/main.scss

```diff
@@ -94,3 +94,7 @@
 
 @import 'overrides/overrides.scss';
 @import 'overrides/print.scss';
+
+.pac-container {
```

Comment:
Not sure if you want me to remove this line, or pac-container class....
I added this z-index b/c the drop-down for cities in the modal was falling behind the modal itself.

Pls let me know...